nck/utils/date_handler.py

```diff
@@ -63,3 +63,24 @@ def get_date_start_and_date_stop_from_date_range(date_range: str) -> Tuple[date,
     """
     current_date = date.today()
     return DEFAULT_DATE_RANGE_FUNCTIONS[date_range](current_date)
+
+
+def build_date_range(start_date, end_date, date_range):
+    """Returns date start and date end adapted if there is a date range.
+
+    Args:
+        start_date (date): start date
```

Comment:
By docstring which doesn't bring value I mean this btw. Your naming is explicit enough as the variables are quite simple here, only the date is interesting but can be mentioned using type hinting.